### ISSUE TITLE
Accessibility: Button does not follow the established design pattern

### ISSUE THREAD

**Issue by jason-me** [stakwork/sphinx-tribes-frontend#1047]
## Issue
The "Sign In" button on the landing page does not follow the established design pattern for button controls. It is not focusable or actionable by the keyboard and does not announce a `button` role. 

## Expected Behavior
- [WCAG 2.2 AA 4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG22/#name-role-value) requires that the `button` role is used.
- [WCAG 2.2 A 2.1.1 Keyboard](https://www.w3.org/TR/WCAG22/#keyboard) requires that interactive controls are focusable and operable with the keyboard.
- Screen reader and keyboard only users expect to be able to navigate to the control and that the proper accessible name and `button` role are announced when interacting with it.
## Acceptance Criteria
- [x] Make this control render as an HTML `button` or add `role="button"` and `tabindex="0"`
- [x] Test with a screen reader JAWS, VoiceOver, NVDA, Orca, Talkback, etc. 
- [x] Ensure the control announces the accessible name which matches the visible label for the control and the  `button` role

_Linked to tracking issue: #456_

**Comment by jason-me** [comment]
PR in pending review status for completion of this issue: https://github.com/stakwork/sphinx-tribes-frontend/pull/1054

**Comment by jason-me** [comment]
Closed by: https://github.com/stakwork/sphinx-tribes-frontend/pull/1054
